website/cue/reference/components/sources/kubernetes_logs.cue

```diff
@@ -42,6 +42,11 @@ components: sources: kubernetes_logs: {
 			"""
 				[Kubernetes](\(urls.kubernetes)) version `\(services.kubernetes.versions)` is required.
 				""",
+			"""
+				This source reads log files from the host, *not* the Kubernetes API. If Vector is running within Kubernetes,
```

Comment:
This first sentence seems duplicative of the source's [description](https://github.com/vectordotdev/vector/pull/16254/files#diff-27a999c10c084357890bce5123fbed6c6cb80f3fc98a70b27b1ebd5b083be21eR9-R10) - what do you think?

Reply:
Maybe we can do `s/host/vector host/`? The description mentions "nodes," but not a specific one. The confusion came from not knowing whether this is a way to collect all logs, or just logs specific to the current host.